src/components/Input/Input.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./input.css";
+import { useFilter } from "../../Context/filter-context";
+
+const Input = ({ inputs }) => {
+  const { dispactherforfilter } = useFilter();
```

Comment:
Use the proper name. I think you expected "dispatcherforfilter" here.

Reply:
Thank you for pointing out typo